### PR TITLE
feature/CIV-2676/response submission FA pay immediately

### PR DESCRIPTION
**Pull request by laperla11**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-2676

https://tools.hmcts.net/jira/browse/CIV-4451

### Change description ###
- This story covers the concept for different scenarios on submit confirmation page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/routes/urls.ts [hunk 1]
```diff
@@ -111,6 +111,7 @@ export const ELIGIBILITY_INFORMATION_FEES_URL = `${BASE_ELIGIBILITY_URL}/informa
 export const ELIGIBILITY_APPLY_HELP_FEES_URL = `${BASE_ELIGIBILITY_URL}/apply-for-help-with-fees`;
 export const ELIGIBLE_FOR_THIS_SERVICE_URL = `${BASE_ELIGIBILITY_URL}/eligible`;
 export const FIRST_CONTACT_CLAIM_REFERENCE_URL =  `${BASE_FIRST_CONTACT_URL}/claim-reference`;
-export const FIRST_CONTACT_CLAIM_PIN_URL =  `${BASE_FIRST_CONTACT_URL}/pin`;
+export const FIRST_CONTACT_CLAIM_PIN_URL = `${BASE_FIRST_CONTACT_URL}/pin`;
```
Comment (GorilaNaranja) [inline src/main/routes/urls.ts:114]: I named it:
export const FIRST_CONTACT_PIN_URL = `${BASE_FIRST_CONTACT_URL}/pin`;

File: src/main/services/features/response/submitConfirmation/submitConfirmationBuilder/admissionSubmitConfirmationContent.ts [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+      data: {
+        text: t('PAGES.SUBMIT_CONFIRMATION.WHAT_HAPPENS_NEXT', {lng: lang}),
+      },
+    },
+  ]; 
+}
+
+export function getFAPayImmediatelyNextSteps(claimId: string, claim: Claim, lang: string): ClaimSummarySection[]{
+  const claimantName = claim.getClaimantName();
+  // TODO : submission day + 5 days
```
Comment (GorilaNaranja) [inline src/main/services/features/response/submitConfirmation/submitConfirmationBuilder/admissionSubmitConfirmationContent.ts:103]: Can you delete TODO as you add `today.setDate(today.getDate() + 5);` right?

File: src/main/views/features/response/submit-confirmation.njk [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      <div class="form-group">
+      {{ govukButton({
+        text: t('PAGES.SUBMIT_CONFIRMATION.GO_TO_ACCOUNT'),
+        href: '/dashboard'
+      }) }}
+      </div>
+        {{ contactUsForHelp(t, '', true) }}
+    </div>
+  </div>
+{% endblock %}
```
Comment (GorilaNaranja) [inline src/main/views/features/response/submit-confirmation.njk:46]: Add empty line at the end of the file

**Review by TatsyMaslova — APPROVED** [review]
amazing job Kadir!!!!!!

**Review by GorilaNaranja — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

**Pull request merged**